tools/WinObjC.Packaging/Package.Default.props

```diff
@@ -18,7 +18,6 @@
     <_ImportLibsFromPackage Include="$(MSBuildThisFileDirectory)\lib\$(TargetOsAndVersion)\$(PlatformTarget)\*.lib"/>    
     <_ImportLibsFromPackage Include="$(MSBuildThisFileDirectory)\lib\$(TargetOsAndVersion)\$(PlatformTarget)\$(Configuration)\*.lib"/>
 
-    <_XamlFilesFromPackage Include="$(MSBuildThisFileDirectory)\xaml\**\*.xaml" />
```

Comment:
See my above comment, 

"Package.Default.props is the default consumption props/targets for middleware using WinObjC"

---
In reply to: [108995058](https://github.com/Microsoft/WinObjC/pull/2372#discussion_r108995058) [](ancestors = 108995058,108994710,108994382,108993288)